### ISSUE TITLE
Server has no SSL certificate

### ISSUE THREAD

**Issue by Artic6** [monitoror/monitoror#422]
How can I bind this to a secure port so its uses HTTPS not HTTP?

I see no option to add a certificate to the configuration?

**Comment by rmdes** [comment]
the simplest way is to have Nginx or Apache or Traefik in front of the app and let nginx or else take care of ssl certificate

**Comment by Artic6** [comment]
Right, I see what you mean so you’re essentially using Apache as a proxy to
do the Secure stuff and then forward the traffic on to the application

Regards,

Lee Croucher
